### PR TITLE
Improve docs of accuracy

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
This is PR for improving docs of functions/links. Related issue: #2182

Modified functions/links:
 - functions/evaluation/accuracy

### CODE REVIEW

File: chainer/functions/evaluation/accuracy.py [hunk 1]
```diff
@@ -59,9 +59,15 @@ def accuracy(y, t, ignore_label=None):
...
     Args:
-        y (Variable): Variable holding a matrix whose (i, j)-th element
-            indicates the score of the class j at the i-th example.
-        t (Variable): Variable holding an int32 vector of ground truth labels.
+        y (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Array whose (i, j, k, ...)-th element indicates the score of
+            the class j at the (i, k, ...)-th sample.
+            The prediction label :math:`\\hat t` is calculated by the formula
+            :math:`\\hat t_(i, k, ...) = argmax_j y(i, j, k, ...)`.
```
Comment (delta2323) [inline chainer/functions/evaluation/accuracy.py]: Remove the underscore after `t`

Comment (delta2323) [inline chainer/functions/evaluation/accuracy.py]: I think we usually do not make `argmax` italic. How about changing it to  `\operatorname{\mathrm{argmax}}` ?

Comment (keisuke-umezawa) [inline chainer/functions/evaluation/accuracy.py]: Thank you for the advice. I will fix them.

**Comment by keisuke-umezawa** [comment]
By the way, the input `t` must be int32 type. I think it is better to hold other integer type. How do you think? If you think the priority of this issue is higher, I will fix them.

**Comment by delta2323** [comment]
I believe it is only because initial version was set and left like this, thinking that we can broaden its dtype afterwards. But I am not perfectly confident.

**Comment by keisuke-umezawa** [comment]
Ok. So, in this PR, I just describe the current functionality.

**Comment by delta2323** [comment]
LGTM

**Pull request merged**